.github/workflows/verify.yaml

```diff
@@ -43,6 +42,7 @@ jobs:
           govet \
           ineffassign \
           golint \
+          gosec \
```

Comment:
Can't we use the `sanity-check` target instead of specifying everything separately?

Reply:
Done.